cmd/ocitools/generate.go

```diff
@@ -78,25 +82,27 @@ var generateCommand = cli.Command{
 			}
 		}
 
-		err := setupSpec(&specgen, context)
+		err := setupSpec(specgen, context)
 		if err != nil {
 			return err
 		}
+		var exportOpts generate.ExportOptions
+		exportOpts.Seccomp = context.Bool("seccomp-only")
 
 		if context.IsSet("output") {
-			output := context.String("output")
-			err = specgen.SaveToFile(output)
+			err = specgen.SaveToFile(context.String("output"), exportOpts)
 		} else {
-			err = specgen.Save(os.Stdout)
+			err = specgen.Save(os.Stdout, exportOpts)
 		}
+
 		if err != nil {
 			return err
 		}
 		return nil
 	},
 }
 
-func setupSpec(g *generate.Generator, context *cli.Context) error {
+func setupSpec(g generate.Generator, context *cli.Context) error {
```

Comment:
Hmmm, Must have accidentally cut it when making the new branch. Thanks for the call out.